vorestation.dme

```diff
@@ -783,6 +783,7 @@
 #include "code\game\machinery\holoposter.dm"
 #include "code\game\machinery\holosign.dm"
 #include "code\game\machinery\igniter.dm"
+#include "code\game\machinery\isatooluserfix_vr.dm"

```

Comment:
Imo better file/path to put the fix in would be "code\modules\nano\interaction\default_vr.dm" for naming/file location consistency.

Reply:
isatooluser as far as I know is defined in machinery, hence why I put it there.

Reply:
Yes, but you are adding/changing a default_can_use_topic proc, which is defined in default.dm (in nano files)